Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:alpine as BUILD
```

Comment:
Do we want to specify go version here? I like just using `golang:alpine` personally, but see that you specify versions elsewhere.